src/integrations/admin/activation-indexation-integration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Admin;
+
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast\WP\SEO\Conditionals\Traits\Admin_Conditional_Trait;
+use Yoast\WP\SEO\Integrations\Cleanup_Integration;
+
+/**
+ * This integration registers a run of the cleanup routine whenever the plugin is activated.
+ */
+class Activation_Indexation_Integration implements Integration_Interface {
+
+	use Admin_Conditional_Trait;
```

Comment:
I would feel better if we didn't use the admin_conditional and loaded this always, much like we do with Cleanup_Integration which similarly uses the deactivation hooks. That way we don't create the new trait either

Reply:
I added this since it uses an activation hook that I guess only `runs` when you active the plugin and you always do that as an admin I thought? Or is my thinking of here?

Reply:
> I guess only runs when you active the plugin and you always do that as an admin I thought?

That's true, but it makes more sense to me to have it "run" every time, makes it more flexible (and it's what we're already for the other setup hooks)